utils/utils.go

```diff
@@ -760,3 +760,37 @@ func getHomeDir() (string, error) {
 func IsDockerCmd(kind CmdType) bool {
 	return (kind == DockerRun || kind == DockerStart || kind == DockerCompose)
 }
+
+func CreateGitIgnore(logger *zap.Logger, path string) error {
+	gitignorePath := path + "/.gitignore"
+	reportEntry := "/reports/"
```

Comment:
Be specific with the entry I think keploy/reports/ works better I guess. Please have a look.

Reply:
I am storing .gitignore file in keploy folder itself as it was mentioned in the issue , So keploy/reports/ entry will not work.

Reply:
cool got it